ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -71,7 +71,7 @@
     // View attributes, defaults are set in init().
     private float textSize;
     private int textColor;
-    private long animationDurationInMillis;
+    private int animationDurationInMillis;
```

Comment:
ah, this should still be a `long` since setting the animation duration programmatically should take in the same type as the animator. We are simply changing the default to an `int` since you can's specify `long` via XML and the default value is 350 anyway so it wouldn't make any difference.